### PR TITLE
iconsextract: Add version 1.47

### PR DESCRIPTION
**Pull request by wenmin92**
[IconsExtract](https://www.nirsoft.net/utils/iconsext.html) - Extract icon/cursor stored in EXE, DLL, OCX, CPL files.

### CODE REVIEW

File: bucket/iconsextract.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "pre_install": "if(!(Test-Path(\"$persist_dir\\iconsext.cfg\"))) { New-Item \"$dir\\iconsext.cfg\" | Out-Null }",
+    "bin": "iconsext.exe",
+    "shortcuts": [
+        [
+            "iconsext.exe",
+            "IconsExtract"
+        ]
+    ],
+    "persist": "iconsext.cfg",
+    "checkver": {
```
Comment (Ash258) [inline bucket/iconsextract.json]:
```suggestion
    "checkver": "IconsExtract v([\\d.]+)",
```

**Comment by Ash258** [comment]
I do not think duplicating manifests is good idea. Nirlauncher is in extras and separate bucket already exists with all tools. Same as JetBrains and SysInternals.

Better to improve quality of nirsoft bucket.

**Comment by wenmin92** [comment]
Although this app is created by _nirsoft_, the `nirsoft` bucket does not contains it.
We cannot search the contained apps by scoop. I searched "iconsectract", but not find, so I PR this app.
I think a whole Nirlauncher is not a good idea, users may not need them all, people will only search some single apps, if them can't search, they didn't know the nirlauncher contains it.

**Comment by Ash258** [comment]
> nirsoft bucket does not contains it

If you are talking about old nirsoft bucket, then lukesampson/scoop#2808

image

> I searched "iconsectract"

Name of app is not `iconsextract`, but `iconsext` (homepage). lukesampson/scoop#3446

**Comment by wenmin92** [comment]
Thanks for these infomation. 
Since this app was already in the https://github.com/MCOfficer/scoop-nirsoft bucket, I will close this PR.

**Pull request closed without merging**